imap_processing/tests/lo/test_lo_l1a_write_cdfs.py

```diff
@@ -63,6 +65,78 @@ def direct_events():
     return [scide1, scide2]
 
 
+@pytest.fixture()
+def science_counts():
+    fake_data_field = namedtuple("fake_packet", ["raw_value", "derived_value"])
+    scicnt1 = ScienceCounts.__new__(ScienceCounts)
+    scicnt2 = ScienceCounts.__new__(ScienceCounts)
+    scicnt1.ccsds_header = CcsdsData(
+        {
+            "VERSION": fake_data_field(0, 0),
+            "TYPE": fake_data_field(0, 0),
+            "SEC_HDR_FLG": fake_data_field(0, 0),
+            "PKT_APID": fake_data_field(705, 705),
+            "SEQ_FLGS": fake_data_field(0, 0),
+            "SRC_SEQ_CTR": fake_data_field(0, 0),
+            "PKT_LEN": fake_data_field(0, 0),
+        }
+    )
+
+    scicnt2.ccsds_header = CcsdsData(
+        {
+            "VERSION": fake_data_field(0, 0),
+            "TYPE": fake_data_field(0, 0),
+            "SEC_HDR_FLG": fake_data_field(0, 0),
+            "PKT_APID": fake_data_field(705, 705),
+            "SEQ_FLGS": fake_data_field(0, 0),
+            "SRC_SEQ_CTR": fake_data_field(0, 0),
+            "PKT_LEN": fake_data_field(0, 0),
+        }
+    )
+    fields = [
+        "START_A",
+        "START_C",
+        "STOP_B0",
+        "STOP_B3",
+        "TOF0",
+        "TOF1",
+        "TOF2",
+        "TOF3",
+        "TOF0_TOF1",
+        "TOF0_TOF2",
+        "TOF1_TOF2",
+        "SILVER",
+        "DISC_TOF0",
+        "DISC_TOF1",
+        "DISC_TOF2",
+        "DISC_TOF3",
+        "POS0",
+        "POS1",
+        "POS2",
+        "POS3",
+        "HYDROGEN",
+        "OXYGEN",
+    ]
+
+    scicnt1.SHCOARSE = np.array([1712787078, 1712787090, 1712787100])
+    scicnt2.SHCOARSE = np.array([1712787150, 1712787170, 1712787200])
+    for field in fields:
+        setattr(scicnt1, field, np.array([0, 0, 0]))
```

Comment:
These test inputs array shapes don't make sense for this data. Need to make them 6x7 and 60x7 arrays